target_postgres/sql_base.py

```diff
@@ -616,9 +616,12 @@ def _serialize_table_record_field_name(self, remote_schema, streamed_schema, pat
             if not mapping is None:
                 return mapping
 
-        raise Exception('Unknown column path: {} for table: {}'.format(
+        raise Exception("Column path: {} for table: {} for the value's JSONSchema {} cannot be found." \
+                        + " This means that the `remote_schema` has no compatible column for this path" \
```

Comment:
This is probably jibberish to most users who haven't dived deep into the code. I even had to think, parse it apart, and look through the code to make sense of it.

Maybe "Column {path} for table {remote_schema['path'][0]} cannot be found in the target database"? Maybe flatten the `path` as well so it legible as a column name if possible?

Reply:
So I think the _path_ more useful here. If we get this exception, we cannot actually tell you what the name of the column is...

Reply:
How about `A compatible column for path {} and JSONSchema {} in table {} cannot be found.`?

Reply:
I think that's better